content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods.md

```diff
@@ -16,33 +16,124 @@ toc: true
 
 > _This document relates to Chainguard Enforce. In order to follow along, you will need access to Chainguard Enforce. You can request access through selecting **Chainguard Enforce for Kubernetes** on the [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs)._
 
-While the `chainctl` approach outlined in our [user onboarding documentation](../chainguard-enforce-user-onboarding/) is a valid and well supported method of installing the Chainguard Enforce Agent, some may prefer alternative methods to better align with their existing Kubernetes workflows.
+There are currently three recommended approaches to installing the Chainguard Enforce Agent. This guide will walk you through each of these approaches. 
 
-We'll refer to these alternative methods as various flavors of Declarative Installs. In other words, the entirety of the Chainguard Enforce Agent resources are defined as static Kubernetes manifests.
+Our [getting started guide](../chainguard-enforce-user-onboarding/) provides more detailed information on how to set up Chainguard Enforce, and this document provides a reference for three alternative methods to align best with your team's existing Kubernetes workflow. 
 
-Static manifests created for Declarative Installs allow for great flexibility for how they're ultimately applied to the cluster, and can be easily adapted to fit most Kubernetes deployment workflows.
+Our first method will be installation with `chainctl`, the command line tool for working with Chainguard products. 
 
-The two examples we'll cover are: "raw" yaml, and a helm chart.
+The second two methods are two approaches to declarative installs, which define the entirety of the Chainguard Enforce Agent resources as static Kubernetes manifests. Static manifests created for declarative installs allow for greater flexibility for how they're ultimately applied to the cluster, and can be adapted to fit most Kubernetes deployment workflows. These two examples we'll be covering are installation via "raw" YAML, and via a Helm Chart.
 
-## Raw YAML
+## Prerequisites
 
-First, we'll fetch the latest YAML representing the Chainguard Enforce Agent's Kubernetes resources:
+You'll need access to Chainguard Enforce (sign up via the [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs)), with a group and cluster already set up.
+
+Regardless of the method you choose, you'll need `chainctl` installed, which you can achieve by following our [How to Install `chainctl` tutorial](/chainguard/chainguard-enforce/how-to-install-chainctl/).
+
+## Install with `chainctl`
+
+To install with `chainctl`, first authenticate into `chainctl` before running a command.
+
+```sh
+chainctl auth login
+```
+
+With your cluster already set up, you'll install the Chainguard Enforce Agent with this `chainctl` command:
+
+```sh
+chainctl cluster install --group=$GROUP_ID --private --context $CLUSTER
```

Comment:
Is it right that `--private` flag is not needed for non-private clusters? Does it matter if it is here as a catchall?

Reply:
Hmm, here's what the flag shows in `chainctl cluster install --help`:
> Kubernetes API endpoint isn't publicly accessible. Cannot be used with managed clusters.

I'm pretty sure this means EKS and GKE clusters won't work with this flag.

Reply:
Ah right, further down here on line 76, so AKS on Azure as well.